lbz/misc.py

```diff
@@ -113,10 +126,3 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
             return default_return
 
     return wrapped
-
-
-def copy_without_keys(data: MutableMapping, *keys: str) -> dict:
```

Comment:
I don't see any value in changing the functions' order here. So if not needed, it would be good to not touch the code at all - just a tip for the future on how to improve reviewers life 😉